assets/src/blocks/event-date/block.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "gatherpress/event-date",
+	"version": "0.1.2",
+	"title": "GatherPress Event Date",
+	"category": "gatherpress",
+	"icon": "clock",
+	"description": "block for selecting dates and times.",
+	"attributes": {
+		"blockId": {
+			"type": "string"
+		}
+	},
+	"supports": {
+		"html": false
+	},
+	"example": {},
+	"textdomain": "gatherpress",
+	"editorScript": "file:./index.js",
+	"editorStyle": "file:./index.css",
+	"style": "file:./style-index.css"
+}
```

Comment:
convert file from spaces to tabs

Reply:
There was an .editorconfig in the assets folder that causing this. I removed it and put one in the root.

Reply:
Hmmmm, maybe JSON is spaces rather than tabs in WP standards... TIL. You can add that back and leave this.
`[{.jshintrc,*.json,*.yml}]
indent_style = space
indent_size = 2`